app/slackimport.go

```diff
@@ -509,9 +516,15 @@ func (a *App) SlackAddChannels(teamId string, slackchannels []SlackChannel, post
 			Type:        sChannel.Type,
 			DisplayName: sChannel.Name,
 			Name:        SlackConvertChannelName(sChannel.Name, sChannel.Id),
-			Purpose:     sChannel.Purpose["value"],
-			Header:      sChannel.Topic["value"],
+			Purpose:     sChannel.Purpose.Value,
+			Header:      sChannel.Topic.Value,
+		}
+
+		// needed to circumvent slacks model for dm chats, or else the messages won't get imported
```

Comment:
nit: `circumvent` has a very different connotation than I expect is meant here. Perhaps you could explain slacks model instead and thus why we use the `Id` as the `Name`?

Reply:
Have reworded the comment to be more precise. The problem was that the way the importer worked for all the other channels was that it matched the messages to the channel names, but DM Channels apparently don't have a name like the other types do. So setting the `Id` as the `Name` here works around this by giving the direct messages a channel name to match against so they get imported too.